### ISSUE TITLE
Header format failure with local dev chain on substrate master

### ISSUE THREAD

**Issue by jnaviask** [polkadot-js/apps#476]
Running in Chrome both locally hosted and via https://polkadot.js.org/apps/#/explorer gives the following error:
```
index.js:128 Error: Error formatting '[object Object]' as 'Header': Expected hex value to convert, found '[object Object]'
    at f (format.js:50)
    at r.map (format.js:69)
    at Array.map (<anonymous>)
    at t.default (format.js:61)
    at t.default (output.js:41)
    at t.default.formatResult (index.js:140)
    at Object.s [as callback] (index.js:103)
    at t.default.onSocketMessageSubscribe.e [as onSocketMessageSubscribe] (index.js:127)
    at WebSocket.onSocketMessage.e (index.js:82)
```

Steps to reproduce:
1. `cargo build` with current substrate master (https://github.com/paritytech/substrate/commit/01ff6f0034f84493db7d8b46b74a997600945678 at time of issue open)
2. `cargo run -- --dev`.
3. Open https://polkadot.js.org/apps/#/explorer. You should see it connect to a node, but no block data.
4. Check site console for error.

**Comment by jacogr** [comment]
Master only supports poc-2. For substrate master support, use https://polkadot.js.org/apps/next/

"next" indicating the "next" branch - will be merged into master as soon as Substrate hits 1.0-beta and Polkadot poc-3 is there.

**Comment by jacogr** [comment]
EDIT: Just to note, you can change the node it connects to under "Settings" - by default it connects to BBQ, but just swapping there connects to a local node. (I have updated master README as well with a note to the link)
